src/app/layouts/BaseLayout.jsx

```diff
@@ -13,6 +14,7 @@ function BaseLayout() {
 			<Route exact path='/register' Component={RegisterPage} />
 			<Route exact path='/favourites' Component={Favourites} />
 			<Route exact path='/user-profile' Component={UserProfilePage} />
+			<Route exact path='/userlocation' Component={UserLocationPage} />
```

Comment:
route names should follow consistent conventions
there is a mix of kebab-case and camelCase